src/cli/cli.cpp

```diff
@@ -3767,6 +3769,34 @@ otError Interpreter::ProcessTxPower(uint8_t aArgsLength, char *aArgs[])
     return error;
 }
 
+otError Interpreter::ProcessUptime(uint8_t aArgsLength, char *aArgs[])
+{
+    OT_UNUSED_VARIABLE(aArgs);
+
+    otError  error = OT_ERROR_NONE;
+    uint32_t now;
+    uint32_t seconds;
```

Comment:
```suggestion
    uint8_t seconds;
```

`uint8_t` is enough for 0 ~ 59 ?